examples/build.gradle.kts

```diff
@@ -32,7 +32,7 @@ dependencies {
     //jafama
     implementation(project(":kmath-jafama"))
     //multik
-    implementation(projects.kmathMultik)
+    implementation(project(":kmath-multik"))
```

Comment:
Why?

Reply:
That's inconsistent. I think there should be a separate PR to replace all `project("...")` with type-safe projects.